docs/Ingress.md

```diff
@@ -543,6 +551,59 @@ spec:
           servicePort: 8000
 ```
 
+Example to create a TLS enabled Ingress configuration
+
+```
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    nginx.ingress.kubernetes.io/affinity: "cookie"
+    nginx.ingress.kubernetes.io/affinity-mode: "persistent"
+    nginx.ingress.kubernetes.io/session-cookie-name: "route"
+    nginx.ingress.kubernetes.io/session-cookie-expires: "172800"
+    nginx.ingress.kubernetes.io/session-cookie-max-age: "172800"
+    nginx.ingress.kubernetes.io/client-body-buffer-size: 100M
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+    nginx.ingress.kubernetes.io/session-cookie-samesite: "true"
+    nginx.ingress.kubernetes.io/session-cookie-path: "/en-US"
+    cert-manager.io/cluster-issuer: selfsigned
+  name: splunk-ingress
+  namespace: default
+spec:
+  rules:
+  - host: shc.example.com
+    http:
+      paths:
+      - path: /en-US
+        pathType: Prefix
+        backend:
+          serviceName: splunk-shc-search-head-service
+          servicePort: 8000
+  - host: hec.example.com
+    http:
+      paths:
+      - path: /services/collector
+        pathType: Prefix
+        backend:
+          serviceName: splunk-idc-indexer-service
+          servicePort: 8088
+  tls:
+  - hosts:
+    - shc.example.com
+    - hec.example.com
+    secretName: operator-tls
+```
+Note:
```

Comment:
See comment above about removing lines 598 - 605 and moving them above the example.

Reply:
Fixed